endtoend/long_minimal_e2e_test.go

```diff
@@ -16,7 +16,7 @@ func TestEndToEnd_Long_MinimalConfig(t *testing.T) {
 	testutil.ResetCache()
 	params.UseMinimalConfig()
 
-	epochsToRun := 20
+	epochsToRun := 100
```

Comment:
This doesn't matter right? It gets set from ENV variable

```suggestion
	var epochsToRun int
```

Reply:
Ah my bad was for testing